### PR TITLE
Updates to the ML taxi fare tutorial

### PR DESCRIPTION
**Pull request by pkulikov**
Main reasons to create this PR:
- Add a note that the `float` type should be used for the fields in data classes.
- Clarify that the **Store** column contains predicted values in case of the regression task (and not for all ML tasks like the current text implies).
- Mention that the **Label** and **Features** columns are the default columns for labels and features, respectively.

### CODE REVIEW

**Review by OliaG — APPROVED** [review]
LGTM

File: docs/machine-learning/tutorials/taxi-fare.md [hunk 1]
```diff
@@ -174,11 +177,11 @@ Notice that the `TripTime` column, which corresponds to the `trip_time_in_secs`
...
+After adding the data to the pipeline and transforming it into the correct input format, you select a learning algorithm (**learner**). The learner trains the model. You chose a **regression** task for this problem, so you use a <xref:Microsoft.ML.Trainers.FastTreeRegressor> learner, which is one of the regression learners provided by ML.NET.
 
 <xref:Microsoft.ML.Trainers.FastTreeRegressor> learner utilizes gradient boosting. Gradient boosting is a machine learning technique for regression problems. It builds each regression tree in a step-wise fashion. It uses a pre-defined loss function to measure the error in each step and correct for it in the next. The result is a prediction model that is actually an ensemble of weaker prediction models. For more information about gradient boosting, see [Boosted Decision Tree Regression](/azure/machine-learning/studio-module-reference/boosted-decision-tree-regression).
 
-Add the following code into the `Train` method following the data processing code added in the previous step:
+Add the following code into the `Train` method following the data processing code added in the preceding section:
```
Comment (JRAlexander) [inline docs/machine-learning/tutorials/taxi-fare.md]: Let's leave this as "previous step", as it's more consistent.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Just one small item, and then I think we will be ready.

File: docs/machine-learning/tutorials/taxi-fare.md (hunk 1 above)
Comment (pkulikov) [reply docs/machine-learning/tutorials/taxi-fare.md]: @JRAlexander fixed!

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**